packages/fxa-content-server/app/images/icon-show-password.svg

```diff
@@ -1 +0,0 @@
-<svg data-name="Layer 2" xmlns="http://www.w3.org/2000/svg" width="55" height="46" viewBox="-16 -13 47 38"><path d="M16 5.7A8.32 8.32 0 0 0 8 0 8.32 8.32 0 0 0 .05 5.7a.86.86 0 0 0 0 .59A8.31 8.31 0 0 0 8 12a8.32 8.32 0 0 0 8-5.7.88.88 0 0 0 0-.6zM8.5 3A1.5 1.5 0 1 1 7 4.5 1.5 1.5 0 0 1 8.5 3zM8 10a6.33 6.33 0 0 1-5.94-4 6.08 6.08 0 0 1 2.5-3A4 4 0 0 0 4 5a4 4 0 0 0 8 0 4 4 0 0 0-.56-2 6 6 0 0 1 2.5 3A6.33 6.33 0 0 1 8 10z"/></svg>
```

Comment:
This one is tricky because we actually DO use it, we just [base64'd/inlined](https://github.com/mozilla/fxa/blob/master/packages/fxa-content-server/app/scripts/templates/partial/show-password.mustache#L2) it.

We probably need an issue made for discussion on the best way to handle our SVGs. We add a hash in the content server so we rarely `<img src="`, but we need to sometimes, and inlining SVGs not only reduces HTTP requests but there's no need to bust any cache on them. It might be ideal to automate something like this:
```
<img src='data:image/svg+xml;utf8,<svg ... > ... </svg>'>
```

... so that way we're still inlining, but but we're not base64'ing (keeping the integrity of the SVG intact and reducing conversions). Regardless I'd say leave this one in here.

Reply:
Big fan of inline SVG. You also have much more flexibility when styling individual <g>roups or <path>s.